pkg/controller/workload/workload_event.go

```diff
@@ -43,6 +43,9 @@ var (
 	ServiceCache map[string]Endpoints = make(map[string]Endpoints)
 )
 
+var SecurityAddDataChannel = make(chan string)
+var SecurityDelDataChannel = make(chan string)
```

Comment:
IIUC, this is to notify workload event. So we can only have one channel, for workload event.

The channel element can be struct{ uid， event}